sources/gdal/large_image_source_gdal/__init__.py

```diff
@@ -872,14 +872,8 @@ def _convertProjectionUnits(self, left, top, right, bottom, width, height,
                 'left and right and at least one of top and bottom is '
                 'specified.')
         if not self.projection:
-            pleft, ptop = self.toNativePixelCoordinates(
-                right if left is None else left,
-                bottom if top is None else top,
-                units)
-            pright, pbottom = self.toNativePixelCoordinates(
-                left if right is None else right,
-                top if bottom is None else bottom,
-                units)
+            pleft, ptop = self.toNativePixelCoordinates(left or right, top or bottom, units)
```

Comment:
reduce complexity with nice use of the or parameter

Reply:
None != 0, so this doesn't actually do the same thing.  Specifically, this is if right is not specified, use left as a fall-back.  You might want a 0 coordinate depending on the crs (e.g., it could be the prime meridian of your reference system).

Reply:
`left is None or right` will do the trick then right ?

Reply:
sorry didn't thought about actual zero value (Flake8 would have complained anyway)